pulsar-functions/api-java/pom.xml

```diff
@@ -49,6 +49,13 @@
       <scope>compile</scope>
     </dependency>
 
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>pulsar-client-admin-original</artifactId>
```

Comment:
@freeznet Can you move the admin API to a separate module (i.e. move them to a `pulsar-client-admin-api` module)? So we don't need to reference an implementation module in the `api-java` module.

Reply:
I have moved apis to module `pulsar-client-admin-api` with package `org.apache.pulsar.client.admin.api`, and also moved the implementation from package `org.apache.pulsar.client.admin.internal` to `org.apache.pulsar.client.admin.impl` to align with pulsar client module. PTAL @sijie if this change is okay, thanks.

Reply:
@freeznet I don't think we need to rename the packages at this moment. For this pull request, we can just move the files to different modules. That would be sufficient for now. Changing the API classes is a breaking change. We need to think more carefully about that.